lib/Client/Client.js

```diff
@@ -1094,6 +1110,54 @@ var Client = (function (_EventEmitter) {
 		}
 
 		/**
+   * The friends that the Client is aware of. Only available after `ready` event has been emitted.
```

Comment:
There seems to be some inconsistent indentation here and on the other doc comments below